style.css

```diff
@@ -1130,3 +1130,7 @@ ul {
   flex-direction: column;
   gap: 10px;
 }
+
+.error-message {
```

Comment:
[OPTIONAL] Good job but you can add some styling to show it's an error. For example, you can use a bigger red font. And you can use set timeout to fade out your error message after some time.
<img width="582" alt="Screenshot 2023-01-07 at 10 00 42 PM" src="https://user-images.githubusercontent.com/25887253/211166436-c41a8251-9cb1-4270-9231-1d873f848317.png">